app/src/main/java/com/appdev/eateryblueandroid/ui/navigation/HomeTabController.kt

```diff
@@ -45,6 +49,17 @@ fun HomeTabController(
                     expandedSectionViewModel = expandedSectionViewModel,
                     hideSection = homeTabViewModel::transitionEateryList
                 )
+            is HomeTabViewModel.State.SearchScreenVisible ->
+                SearchingScreen(
```

Comment:
this is good stuff here

Reply:
we can either pass in eateries or the homeViewModel here to searching screen (see below comment)

Reply:
eateries is preferable (makes searchscreen and homeviewmodel more independent)

Reply:
Here is a [link](https://www.nexmobility.com/articles/android-app-using-mvvm-architecture.html) for what I am talking about. Essentially, the data is only called once in the repository which is a singleton class. The repository is referenced in the view models. I think this will make getting the eateries easier, but will also help with maintainability. On Volume, we are running into data being out of sync with the UI, and I will be changing the architecture to a repository